src/sentry/options/defaults.py

```diff
@@ -1771,7 +1771,7 @@
 )  # ms
 register(
     "performance.issues.http_overhead.http_request_delay_threshold",
-    default=500,
+    default=250,
```

Comment:
Making it easier trigger overhead issues, http 2.0 is almost a decade old and can easily parallelize --  I think we should make these more obvious if customers are experiencing these needless delays.